apple/macos.doc.bzl

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Bazel rules for creating macOS applications and bundles."""
+"""# Bazel rules for creating macOS applications and bundles."""
```

Comment:
intentional?

Reply:
yes, that's just correcting macos to look like tvos and watchos, was a mistake in the previous PR